processing_handler.go

```diff
@@ -184,12 +184,12 @@ func handleProcessing(reqID string, rw http.ResponseWriter, r *http.Request) {
 
 	checkTimeout(ctx)
 
-	if len(conf.SkipProcessingFormats) > 0 {
+	po := getProcessingOptions(ctx)
+	if len(conf.SkipProcessingFormats) > 0 || len(po.SkipProcessingFormats) > 0 {
 		imgdata := getImageData(ctx)
-		po := getProcessingOptions(ctx)
 
 		if imgdata.Type == po.Format || po.Format == imageTypeUnknown {
-			for _, f := range conf.SkipProcessingFormats {
+			for _, f := range append(conf.SkipProcessingFormats, po.SkipProcessingFormats...) {
```

Comment:
I'm not sure, is `append` is good enough despite possible duplication? @DarthSim